psydac/feec/struphy_example.py

```diff
@@ -0,0 +1,70 @@
+from mpi4py import MPI
```

Comment:
I think we have a more complete example in our unit tests, right? So we can remove this old file.